data/types/base/base_test.go

```diff
@@ -48,26 +48,26 @@ import (
 // TODO_DATA: Need to find another way to test base not using sample/sub
```

Comment:
I think the best thing would be to create a concrete "test" type that uses base and test base through that. One of the TODOs that needs to be addressed before prd.